src/orders/components/OrderListDatagrid/datagrid.test.ts

```diff
@@ -205,3 +211,37 @@ describe("useGetCellContent", () => {
     expect((getCellContent([0, 1], contentOpts) as TextCell).data).toBe("");
   });
 });
+
+describe("getPaymentCellContent", () => {
+  it("should return Fully Paid when payment status is PAID", () => {
+    // Arrange
+    const data = {
+      paymentStatus: "PAID" as PaymentChargeStatusEnum,
+    } as RelayToFlat<NonNullable<OrderListQuery["orders"]>>[number];
```

Comment:
suggestion: would be good to keep that long ad repeated types in one place `RelayToFlat<NonNullable<OrderListQuery["orders"]>>[number]`